### ISSUE TITLE
is the "Using Multiple Keys" section still accurate?

### ISSUE THREAD

**Issue by mlysle** [drduh/YubiKey-Guide#409]
I setup two yubikeys with the same PGP keys. I tested out switching between them and it works seamlessly without any manual intervention other than entering the new PIN. Then is the "Using Multiple Keys" section outdated or have I done something wrong?

**Comment by Paraphraser** [comment]
Well, I've had similar experiences - switching identically-provisioned keys without fuss when I've expected pushback.

I've also had the situation where the same key used on the same Mac twice in succession has told me that I need to insert the key which is actually already inserted, and no amount of unplug/replug will fix it.

Overall though, mostly the only time I get pushback is the "Keys A then B on same host" scenario.

Bottom line, in my experience it mostly just works but, when it doesn't, the command  in the Multiple Keys section is a pretty good fix.
